raiden/transfer/events.py

```diff
@@ -54,12 +54,16 @@ def __ne__(self, other):
         return not self.__eq__(other)
 
     def to_dict(self) -> typing.Dict[str, typing.Any]:
-        return {
+        result = {
             'channel_identifier': self.channel_identifier,
             'token_address': to_checksum_address(self.token_address),
             'token_network_identifier': to_checksum_address(self.token_network_identifier),
             'balance_proof': self.balance_proof,
+
         }
+        if self.balance_proof:
+            result['balance_hash'] = serialization.serialize_bytes(self.balance_proof.balance_hash)
```

Comment:
Not really, we add it to `to_dict` because we want to filter based on the balance hash in the DB. But we already restore the balance_proof so we don't need to restore the balance hash.